app/scripts/metamask-controller.js

```diff
@@ -1339,6 +1346,17 @@ export default class MetamaskController extends EventEmitter {
 
     ///: END:ONLY_INCLUDE_IF
 
+    ///: BEGIN:ONLY_INCLUDE_IF(notifications)
+    // Notification Controllers
+    this.authenticationController = new AuthenticationController({
```

Comment:
Not a big deal question: why is the controller named as `authenticationController` instead of `notificationController` ?

Reply:
Great Q.

We will have a set of controllers that the notifications flag will contain - each with a set of responsibilities.
This is an initial PR with just this controller (was recommended to split into multiple PRs to reduce PR size).